### ISSUE TITLE
Fog changes what's occluded when turning

### ISSUE THREAD

**Issue by halvorgb** [JBustos22/oDFe#41]
Hey I like fog and noticed this problem. I think it's a bug (or shortcoming or technical limitation) in quake 3 originally, as I have downloaded OpenArena and observed the same effect there.

This is a demonstration, at certain angles the effect can be quite jarring: https://streamable.com/2zfw98

**Comment by halvorgb** [comment]
For reference, this is the shader that is used:

```
textures/hgb-mint/fog1024
{
	fogParms ( 0.87 0.967 0.776 ) 1024	//Normalized RGB, distance to opaque

	surfaceparm fog		//Must be used
	surfaceparm nolightmap
	surfaceparm nonsolid
	surfaceparm trans

	q3map_fogDir ( 0 0 -180 )

	qer_editorImage textures/sfx/fog_grey.tga
	qer_trans 0.4
}
```

**Comment by ghost** [comment]
Yeah, this is a well known bug and hard to fix, I think. Though, actually it's a problem with the FOV (field-of-view) which 'breaks' the correct view distance on the sides of your fov. The higher your cg_fov value is, the worse it gets.
